### PR TITLE
Bug 1214869 - [music][nga] Tiles view shouldn't show artist/album nam…

### PR DESCRIPTION
**Pull request by jimporter**
…e if we have real album art

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4198be4ee89faaaab7d3a7e7d8220db003ccac8d

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5aeae9601af8be2f16c5ba43bed9cb79baf7b465

File: apps/music/views/home/view.css [hunk 1]
```diff
@@ -43,10 +43,10 @@ body[data-search="true"] {
...
 }
 
-.tile:nth-child(6n+1):before,
-.tile:nth-child(6n+1):after,
-.tile:nth-child(6n+4):before,
-.tile:nth-child(6n+4):after {
+.tile[data-show-title]:before,
+.tile[data-show-title]:before,
+.tile[data-show-title]:after,
+.tile[data-show-title]:after {
```
Comment (justindarc) [inline apps/music/views/home/view.css]: You have 2 before's and 2 after's

File: apps/music/views/home/view.css [hunk 2]
```diff
@@ -66,13 +66,13 @@ body[data-search="true"] {
   white-space: nowrap;
 }
 
-.tile:nth-child(6n+1):before,
-.tile:nth-child(6n+4):before {
+.tile[data-show-title]:before,
+.tile[data-show-title]:before {
```
Comment (justindarc) [inline apps/music/views/home/view.css]: You only need one `:before` now

File: apps/music/views/home/view.css [hunk 3]
```diff
@@ -66,13 +66,13 @@ body[data-search="true"] {
...
   content: attr(data-artist);
 }
 
-.tile:nth-child(6n+1):after,
-.tile:nth-child(6n+4):after {
+.tile[data-show-title]:after,
+.tile[data-show-title]:after {
```
Comment (justindarc) [inline apps/music/views/home/view.css]: You only need one `:after` now

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4c03b04fa198043b99ba6063f7ca7e14ee0bc41f

File: apps/music/views/home/view.css [hunk 4]
```diff
@@ -11,53 +11,49 @@ body[data-search="true"] {
...
+  border-bottom: solid 0.1rem var(--background);
   box-sizing: border-box;
   display: block;
   float: inline-start;
   text-align: match-parent;
   position: relative;
-  width: 33.3vw;
-  height: 33.3vw;
+  width: calc(100vw / 3);
+  height: calc(100vw / 3);
```
Comment (justindarc) [inline apps/music/views/home/view.css]: Please just add some more 3's instead of using a calc() here.

File: apps/music/views/home/view.css [hunk 5]
```diff
@@ -11,53 +11,49 @@ body[data-search="true"] {
...
   overflow: hidden;
 }
 
+.tile:nth-child(3n+1) {
+  width: calc(100vw * 2 / 3);
+  height: calc(100vw * 2 / 3);
```
Comment (justindarc) [inline apps/music/views/home/view.css]: Same thing here. Just add some 6's to the decimal.

File: apps/music/views/home/view.css [hunk 6]
```diff
@@ -90,6 +95,6 @@ body[data-search="true"] {
   transition: opacity 250ms;
 }
 
-.tile > img.loaded {
+.tile.loaded > img {
```
Comment (justindarc) [inline apps/music/views/home/view.css]: +1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a76e03a2998dd4a1d11277901acb06c59eb26905

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8bac86ed3e267cf8ca14497cb04f15c499442201

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=67821fff18bbd438d1672b9ccb0634eacf8a9f2d

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=fe09b26d3ef3e8e1ee0ae2e412bbc11bb9698ba4

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=649d4ff0f2b0c0af9df5bd7f72d938c633ea15e3

**Pull request merged**